src/data/fetcher.js

```diff
@@ -23,7 +23,7 @@
 
 //@flow
 
-const server_ip = process.env.REACT_APP_FLOWER_SERVER_IP || "0.0.0.0";
+const server_ip = process.env.REACT_APP_FLOWER_SERVICES || "127.0.0.1";
```

Comment:
As far as I remember, this "127.0.0.1" created a few problems, but maybe because I had a very specific firewall setup. 

Have you tried hosting everything on localhost, and see if it works?

Reply:
That constant is used only to indicate the address of flower services, it is not a bind address. I think it's more appropriate to use the loopback address as a fallback instead of the generic '0.0.0.0', but it should be indifferent